source/Calamari.Common/Features/Scripting/Bash/Bootstrap.sh

```diff
@@ -46,7 +46,7 @@ __mask_sensitive_value $sensitiveVariableKey
 # -----------------------------------------------------------------------------
 function decrypt_variable
 {
-	echo $1 | openssl enc -a -A -d -aes-128-cbc -nosalt -K $sensitiveVariableKey -iv $2
+	echo $1 | openssl enc -a -A -d -aes-256-cbc -nosalt -K $sensitiveVariableKey -iv $2
```

Comment:
There shouldn't be any compatibility issues with the increased key length since the ciphers were added at the same time:

https://github.com/openssl/openssl/commit/41450b27f2410f70058eb2e38ec6dffa43f7c7fe